plugin-gestures/src/main/java/com/mapbox/maps/plugin/gestures/GesturesPluginImpl.kt

```diff
@@ -429,28 +429,34 @@ class GesturesPluginImpl : GesturesPlugin, GesturesSettingsBase, MapStyleObserve
     /**
      * Called when an on down gesture was detected.
      */
-    override fun onDown(motionEvent: MotionEvent): Boolean {
+    override fun onDown(motionEvent: MotionEvent?): Boolean {
       return true
     }
 
     /**
      * Called when an on single tap up gesture was detected.
      */
-    override fun onSingleTapUp(motionEvent: MotionEvent): Boolean {
+    override fun onSingleTapUp(motionEvent: MotionEvent?): Boolean {
       return handleSingleTapUpEvent()
     }
 
     /**
      * Called when an on single tap up confirmed gesture was detected.
      */
-    override fun onSingleTapConfirmed(motionEvent: MotionEvent): Boolean {
+    override fun onSingleTapConfirmed(motionEvent: MotionEvent?): Boolean {
+      if (motionEvent == null) {
+        return false
+      }
       return handleClickEvent(motionEvent.toScreenCoordinate())
     }
 
     /**
      * Called when an on double tap gesture was detected.
      */
-    override fun onDoubleTapEvent(motionEvent: MotionEvent): Boolean {
+    override fun onDoubleTapEvent(motionEvent: MotionEvent?): Boolean {
+      if (motionEvent == null) {
+        return false
+      }
       return if (handleDoubleTapEvent(motionEvent, doubleTapMovementThreshold)) {
         true
       } else super.onDoubleTapEvent(motionEvent)
```

Comment:
Should super be called even if it's null? If so, maybe handle the null inside `handleDoubleTapEvent(...)`

Reply:
Hard to say, those cases should not really happen at all - haven't been able to reproduce locally even one single time. Seems it's either buggy implementation of some vendors or Android version.
Also, we've only had reports about such cases for the onFling gesture.
That said, I'd probably left it as is, ignoring such gestures completely.

Reply:
Also, `super` here is just simple instance of listener, it does nothing https://github.com/mapbox/mapbox-gestures-android/blob/master/library/src/main/java/com/mapbox/android/gestures/StandardGestureDetector.java#L98